src/pal/src/file/path.cpp

```diff
@@ -1491,12 +1467,12 @@ SearchPathW(
     
             PathLength = pPathEnd-pPathStart;
     
-            if (PathLength+FileNameLength+1 >= MAX_LONGPATH) 
+            if (PathLength+FileNameLength+1 >= MAX_LONGPATH)// Check if we have to do anything special 
             {
                 /* The path+'/'+file length is too long.  Skip it. */
                 WARN("path component %.*S is too long, skipping it\n", 
                      (int)PathLength, pPathStart);
-                continue;
+                //continue;
```

Comment:
Do we still know for sure whether we are going to skip the segment, if so i will un-comment it